src/utils.ts

```diff
@@ -478,3 +478,14 @@ export function datetimeString(d: Date): string {
 export function isCloudEnvironment() {
   return !!process.env.CODESPACES;
 }
+
+/**
+ * Returns the projectId from a channel name string.
```

Comment:
This is a little too specific of a description for the global `utils` folder, and I'm not sure it warrants being a util given the simplicity of it. Consider refactoring this as a local function in the clone command where it's used.